src/Raven.Server/RavenServer.cs

```diff
@@ -1290,7 +1290,8 @@ private async Task<byte[]> RenewLetsEncryptCertificate(CertificateUtils.Certific
             }
             catch (Exception e)
             {
-                throw new InvalidOperationException("Failed to validate user's license as part of Let's Encrypt certificate refresh", e);
+                string userLicenseMessage = license is null ? ", user's license was not found" : $" with license id: {license.Id}";
```

Comment:
id => ID
instead of doing `:` just wrap it in `''` 

```
Failed to validate user's license '{license.Id ?? "N/A"}' as part of Let's Encrypt certificate refresh.
```

Reply:
done, I left the userLicense string because you cant null collapse different types of objects.